plot.go

```diff
@@ -87,11 +87,11 @@ func New() (*Plot, error) {
 	if err != nil {
 		return nil, err
 	}
-	x, err := makeAxis()
+	x, err := makeAxis(false)
```

Comment:
Personally, I don't like passing a bool here. At the callsite, it's impossible to tell what the bool means. I'd prefer a const:
```
type axisLayout bool
const (
    verticalAxis axisLayout = true
    horizontalAxis axisLayout = false
}
```
or something.

Another option is to leave the XAlign and YAlign as their zero value and set them here at the callsite, since that is the first place where we know how to align the text.